### DISCUSSION TITLE
Issues loading jspsych experiment on Cognition.run

### DISCUSSION THREAD

**Discussion by DylanEKirsch** [jspsych/jsPsych#2967] [Q&A]
Hello, I am very interested in using cognition.run to run a jspsych experiment but am having trouble loading the task. When I try to upload the javascript file for the task, I get the following error: "You don't need to upload JsPsych library. Cognition loads it for you." 
I am wondering what the best way is to load a jspysch experiment?

Thank you so much!

**Comment by ChristopheBossens** [comment]
Are you copy-pasting the whole html file, including the script tags that include the jsPsych library?

On cognition.run you only need to add the javascript part that you add beneath the body section (i.e., the part you have written yourself).
